.github/workflows/deploy.yml

```diff
@@ -80,6 +80,12 @@ jobs:
     - name: Install 7Zip PowerShell Module
       shell: powershell
       run: Install-Module 7Zip4PowerShell -Force -Verbose
+    - name: Download Android Libs
```

Comment:
I think it should be done inside run_android.yaml workflow. And no need to download anything here